### PR TITLE
Added example to read labels from external file

### PR DESCRIPTION
**Pull request by dmenne**
I often read in labels from an external file, e.g. when data come in database with a separate descriptor table. An example was added to show the use of setNames (which is little known and has a bit of a strange parameter sequence).

### CODE REVIEW

File: R/var_label.R [hunk 1]
```diff
@@ -30,6 +30,21 @@
...
+#' description = data.frame(
+#'   name = c(
+#'     "Sepal.Length", "Sepal.Width", "Petal.Length", "Petal.Width",
+#'     "Something"),
+#'   label = c(
+#'     "Sepal length", "Sepal width",  "Petal length", "Petal width",
+#'     "something")
+#' )
+#' var_labels = setNames(as.list(description$label), description$name)
+#' iris_labelled = set_variable_labels(iris, .labels= var_labels, .strict=FALSE)
```
Comment (larmarange) [inline R/var_label.R:45]: please respect tidyverse style, i.e. adding a space before and after =

Comment (larmarange) [reply R/var_label.R:45]: Would also prefer the syntaxe

`iris %>% set_variable_labels(.labels = var_labels, .strict = FALSE)`

**Comment by larmarange** [comment]
Thank your for your proposal. It appears that the example fails with older version of R. 
cf. https://github.com/larmarange/labelled/pull/81/checks

I will try to investigate why

File: R/var_label.R [hunk 2]
```diff
@@ -30,6 +30,21 @@
...
+#' # Set labels from dictionary, e.g. as read from external file
+#' # One description is missing, one has no match
+#' description = data.frame(
+#'   name = c(
+#'     "Sepal.Length", "Sepal.Width", "Petal.Length", "Petal.Width",
+#'     "Something"),
+#'   label = c(
+#'     "Sepal length", "Sepal width",  "Petal length", "Petal width",
+#'     "something")
+#' )
```
Comment (larmarange) [inline R/var_label.R:43]: Please add `stringsAsFactors = FALSE` for R 3.5 and 3.6

**Comment by larmarange** [comment]
OK, I understood the issue.  With R 3, you need to specify `stringsAsFactors = FALSE` when building the data.frame

**Comment by dmenne** [comment]
Hi, I also wanted to use %>%, but then I found that there was a similar example in the non-dplyr section. Your choice. When used in the dplyr section, we could also avoid the historic stringsAsFactors and use tibble(...).

Your choice, please decide

**Comment by larmarange** [comment]
> Hi, I also wanted to use %>%, but then I found that there was a similar example in the non-dplyr section. Your choice. 

I think more educative to use %>% when using the dplyr syntax

> When used in the dplyr section, we could also avoid the historic stringsAsFactors and use tibble(...).

Yes, use tibble()

THANKS

**Comment by dmenne** [comment]
I am somewhat lost what happened with the latest pull request. Why is no build action triggered for the tibble version?

**Review by larmarange — APPROVED** [review]

**Comment by larmarange** [comment]
Thanks it has been merged

**Pull request merged**